docs/source/azure/examples/vmwithpublicip.rst

```diff
@@ -9,5 +9,3 @@ Find Virtual Machines with Public IP address
          filters:
           - type: network-interface
             key: 'properties.ipConfigurations[].properties.publicIPAddress.id'
-            op: eq
-            value: not-null

```

Comment:
Does this work? I thought you needed `value: not-null` and just no `op`

Reply:
That would make more sense. I misunderstood yours and kapil's comments about that. I'll add it back in. Looks like the workers crashed anyway :)